03week/towersOfHanoi.js

```diff
@@ -13,29 +13,67 @@ let stacks = {
   c: []
 };
 
+const stackA=stacks.a;
+const stackB=stacks.b;
+const stackC=stacks.c;
+const stacksArr=[stackA, stackB, stackC];
+// Create an array that houses all stacks so a forEach function can be applied(shortens code).
+
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
 function movePiece() {
-  // Your code here
+
+     let startStack.pop();
+     let endStack.push(startStack);
+  //Write a function that allows you to move the blocks(pop/push).
+
+    }
+
+})
+
+});
+
+movePiece();
 
 }
 
 function isLegal() {
-  // Your code here
+
+  forEach (i = 0; i < stacksArr.length-1; i++) {
```

Comment:
A loop here is unnecessary, you could just check the end piece of the start stack vs the end piece of the end stack. Also you don't need to pop anything in an isLegal function. You should only return a true/false value if the move is legal.